osu.Game.Rulesets.Mania/Objects/Drawables/DrawableNote.cs

```diff
@@ -73,5 +100,12 @@ public virtual bool OnPressed(ManiaAction action)
         public virtual void OnReleased(ManiaAction action)
         {
         }
+
+        private void updateSnapColour()
+        {
+            Colour = configColourCodedNotes.Value
```

Comment:
Sounds like you would want to set this to `AccentColour.Value` instead, rather than the actual drawable colour.

Reply:
Have you tested that this doesn't change any behaviours?

Reply:
I haven't but thought it's the correct direction for colouring drawable hit objects properly, rather than going for `Drawable.Colour`.

Reply:
I just tried it out, right now `AccentColour` doesn't affect the notes of the head and tail (`DefaultNotePiece`) but does affect with the body of a hold note (`DefaultBodyPiece`)

I could change `DefaultNotePiece` to use `AccentColour` but it will also affect the column colors of the hold notes which are currently set in `Stage.cs` ([this code](https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Mania/UI/Stage.cs#L40-L45)). Maybe we can add a second accent color value so we change them separately?